src/components/Modal/Modal.js

```diff
@@ -19,7 +19,7 @@ export default class Modal extends Component {
     onRequestClose: PropTypes.func,
     id: PropTypes.string,
     modalHeading: PropTypes.string,
-    modalLabel: PropTypes.string,
+    modalLabel: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
```

Comment:
Did you mean `PropTypes.node` instead...?

Reply:
Thats an even better solution, thanks!